tasm-staff-editor/src/components/EditorView/MediaType.js

```diff
@@ -23,7 +23,7 @@ export default function MediaType({ mediaType, mediaLink, onChange }) {
         <label htmlFor="media-type" className="mr-2 text-white">
           Media Type:
         </label>
-        <select name='mediaType' id="media-type" value={mediaType} onChange={onChange} className='text-darkBlue flex-1 rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-lightBlue-600'>
+        <select name='mediaType' id="media-type" value={mediaType} onChange={onChange} className='text-darkBlue rounded-md mt-1 border-0 px-2 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-red'>
```

Comment:
Would like to see the properties spread out more like in ExhibitTitle.js

Reply:
This is the only place I can remember seeing anything I'd like to change before merge.